includes/wc-functions.php

```diff
@@ -1265,3 +1265,104 @@ function dokan_modify_vendor_order_counts( $counts ) {
 }
 
 add_filter( 'wp_count_posts', 'dokan_modify_vendor_order_counts', 10, 1 );
+
+if ( ! function_exists( 'dokan_shipping_custom_split_shipping_packages' ) ) {
```

Comment:
@shohag121 instead of adding new functions, can you please add a new class like SplitShipping and move the codebase there?